validator/restapi/src/main/java/tech/pegasys/teku/validator/restapi/ValidatorRestApi.java

```diff
@@ -51,7 +55,7 @@ public static RestApi create(final ValidatorRestApiConfig config) {
         .exceptionHandler(
             BadRequestException.class,
             (throwable, url) -> new HttpErrorResponse(SC_BAD_REQUEST, throwable.getMessage()))
-        .endpoint(new GetKeys())
+        .endpoint(new GetKeys(validatorKeys))
```

Comment:
this isn't really ideal, the list might change over time...

we could potentially pass getOwnedValidators, but a static set will not handle the list changing over time...